client/scss/components/_promo.scss

```diff
@@ -162,14 +173,29 @@
   margin-bottom: 0.9em;
   transition: color 400ms ease;
 
-  @include container-query(('l': ('12'))) {
+  @include container-query(('xl': ('12'))) {
     @include font('WB3');
   }
 
   .promo:hover &,
   .promo:focus & {
     color: color('action-green-2');
   }
+
+  .promo--standalone & {
+    @include respond-to('medium') {
+      @include font('WB4');
```

Comment:
The way this type scales isn't linear with the width of the screen, but based on how the component is laid out – i.e. the font size starts out larger at smaller screen widths, gets smaller at medium-sized screen widths and large again at large screen widths. This particular behaviour probably isn't re-usable enough to be baked into a class.